### DISCUSSION TITLE
Problems with smooth gray matter in occipital and temporal  region of headmodel

### DISCUSSION THREAD

**Discussion by MjSteinecker** [simnibs/simnibs#375] [7.Creating head models]
Hey everyone, so I am new to this field and have problems with my headmodel. I only have a T1w MRI-scan and use the charm function with forceqform and noneck. 

The gray matter at the back of the model and on some temporal region are flat with no visbible gryi and sulci.
![frontalView](https://github.com/simnibs/simnibs/assets/97512684/d29d464e-f30d-4f2b-bc69-282a525d3993)
![occipitalView](https://github.com/simnibs/simnibs/assets/97512684/5bac0887-f737-4ea3-9373-3b1b1d6ebce7)

Additionally when checking the segmentation, the mask is in the lower left corner and not over the scan. 
![MaskShift](https://github.com/simnibs/simnibs/assets/97512684/3e79250e-35fe-4cd2-a7fe-3ea69eda07b1)
Is it possible to fix this model or does it make more sense to record a new MRi-scan +T2? Thanks beforehand :)

**Comment by oulap** [comment]
Hi,

The occipital region is tricky to model accurately using only a volume segmentation. If you want to improve the surfaces, you can try first running freesurfer on this subject and then re-running charm with the `--fs-dir` flag followed by the path to the freesurfer directory. 

I'm not sure why the visualization in the html looks weird, but the gray matter segmentation looks fairly good so it seems that the segmentation worked out even though the visualization is off. That could be an orientation issue. You can check if the input scan and the segmentation output from charm overlap using e.g., freeview or fsleyes. 

Finally, having a T2w scan would be the optimal case, but I would try running freesurfer first and recreating the mesh using those surfaces.
